### PR TITLE
Adding prometheus metrics to inv_mq_service

### PR DESCRIPTION
**Pull request by Jason-RH**
Initial commit. We can discuss other metrics we would like to add.

### CODE REVIEW

File: inv_mq_service.py [hunk 1]
```diff
@@ -59,6 +59,7 @@ def initialize_thread_local_storage(operation_message):
 def main():
     config_name = os.getenv('APP_SETTINGS', "development")
     application = create_app(config_name, start_tasks=False)
+    metrics.start_http_server(9126)
```
Comment (dehort) [inline inv_mq_service.py]: I think we should import the start_http_server method directly in the inv_mq_service.py file.

File: lib/metrics.py [hunk 2]
```diff
@@ -1,11 +1,13 @@
-from prometheus_client import Counter, Summary
+from prometheus_client import start_http_server, Counter, Summary
 
 host_dedup_processing_time = Summary("inventory_dedup_processing_seconds",
                                      "Time spent looking for existing host (dedup logic)")
 new_host_commit_processing_time = Summary("inventory_new_host_commit_seconds",
                                           "Time spent committing a new host to the database")
 update_host_commit_processing_time = Summary("inventory_update_host_commit_seconds",
                                              "Time spent committing a update host to the database")
+ingress_message_parsing_time = Summary("ingress_message_parsing_seconds",
```
Comment (dehort) [inline lib/metrics.py]: I'd like to prefix the name with "inventory_".

What do you think about moving this metric into app/queue/metrics.py?  I think it'd be cleaner to move the metrics closer to where they are used.  I don't think we've done a great job with this in the past, but I think its a good idea.  What do you think?

Comment (Jason-RH) [reply lib/metrics.py]: Yeah I'm fine with this! I was also considering having a separate metrics.py

File: inv_mq_service.py (hunk 1 above)
Comment (Jason-RH) [reply inv_mq_service.py]: :+1:

**Pull request merged**